mediapipe/tasks/cc/genai/inference/c/llm_inference_engine.h

```diff
@@ -15,9 +15,16 @@
 #ifndef MEDIAPIPE_TASKS_GENAI_INFERENCE_C_LLM_INFERENCE_ENGINE_H_
 #define MEDIAPIPE_TASKS_GENAI_INFERENCE_C_LLM_INFERENCE_ENGINE_H_
 
+#ifdef __cplusplus
```

Comment:
The other platforms seem to be using this as a C++ file. This is needed for interoperability with the Swift APIs.